istioctl/cmd/istioctl/convert/testdata/v1alpha3/merged-gateway.yaml.golden

```diff
@@ -38,7 +38,7 @@ spec:
         prefix: /foo/
     route:
     - destination:
-        host: myservice-service
+        host: myservice-service.default.svc.cluster.local
```

Comment:
same question here. Thought it's not required to specify the full name in default namespace

Reply:
this is the generated route rule. Using FQDN maintains clarity. The short name stuff has proven to be very ambiguous.